README.md

```diff
@@ -32,7 +32,8 @@ You can manually track more aspects of your app and system using the API describ
 
 
 
-## Supported Node.JS versions 
+## Supported Node.JS versions
+> *Note:* Versions of Node.JS that are not supported by the [JS Azure SDK](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) will not support AAD authentication.
```

Comment:
This is a little bit confusing, we need azure/identity package to be included to use the feature, as customer need to pass an instance of a Credential object, so this extra note could cause more confusion than helping here.

Reply:
I'll reword to make it clearer what the exception to our support for older versions really is.

Reply:
I moved the note to the aadCredential description. Hopefully that makes it clearer that old versions of Node don't impact using any other scenario with the SDK.